src/legacy/core_plugins/kibana/public/discover/top_nav/open_search_panel.js

```diff
@@ -37,15 +38,17 @@ export class OpenSearchPanel extends React.Component {
 
   renderMangageSearchesButton() {
     return (
-      <EuiButton
+      <EuiButtonEmpty
+        iconSide="left"
+        iconType="gear"
```

Comment:
Icon not necessary for this button and if it's moved to the footer, just make it a regular fill button.